dashboard/config/routes.rb

```diff
@@ -351,6 +351,7 @@ module OPS
     get 'teacher_application/thanks', to: 'teacher_application#thanks'
 
     get 'facilitator_program_registration', to: 'facilitator_program_registration#new'
+    get 'regional_partner_program_registration', to: 'regional_partner_program_registration#new'
```

Comment:
yep, this is the route that will serve the form; the other one handles saving the API request